src/presentation/layouts/ThreeColsLayout.vue

```diff
@@ -32,6 +32,7 @@
 .main-content {
   width: var(--layout-content-width);
   padding: 0 var(--spacing-ml);
+  box-sizing: content-box;
```

Comment:
why `content-box`?

Reply:
for now width of main content layout is fixed and should not be rescaled (without content box main layout would be 700 px, with this box sizing 732 px as it should be)